src/ResourceManager/Insights/Commands.Insights/Autoscale/GetAzureRmAutoscaleSettingCommand.cs

```diff
@@ -37,7 +37,7 @@ public class GetAzureRmAutoscaleSettingCommand : ManagementCmdletBase
         /// </summary>
         [Parameter(ParameterSetName = GetAzureRmAutoscaleSettingParamGroup, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The resource group name")]
         [ValidateNotNullOrEmpty]
-        public string ResourceGroup { get; set; }
+        public string ResourceGroupName { get; set; }
```

Comment:
@gucalder same comment about adding an alias to `ResourceGroup`

Reply:
Done